cmd/mesh/testdata/profile-dump/output/all_off.yaml

```diff
@@ -32,10 +32,10 @@ gateways:
         hpaSpec:
           maxReplicas: 5
           metrics:
-          - resource:
-              name: cpu
-              targetAverageUtilization: 80
-            type: Resource
+            - resource:
```

Comment:
how did these come to be indented? i just want to make sure it's not a manual change that would be reverted next time we regenerate.